src/components/submission/Submission.jsx

```diff
@@ -1,18 +1,68 @@
-import React, { useContext, useEffect } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { AppContext } from "../../contexts/AppContext";
 
 /**
  * Submission
  */
 function Submission() {
   const {
+    apiSaveEndpoint,
     tourTitle,
     tourItems,
     tourDescription,
     validityIssues,
     setValidityIssues,
     limits,
+    isSaving,
+    setIsSaving,
   } = useContext(AppContext);
+  const [saveResponse, setSaveResponse] = useState(null);
+
+  const handleSave = () => {
+    setIsSaving(true);
+    setTimeout(async () => {
+      try {
+        // Post to the API
+        const res = await fetch(`${apiSaveEndpoint}`, {
+          method: "POST",
+          headers: {
+            Accept: "application/json",
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            title: tourTitle,
+            description: tourDescription,
+            artworks: tourItems.map((item) => ({
+              objectNote: item.note,
+              ...item,
```

Comment:
Look into whether this duplicates the note field

Reply:
You were correct, I've fixed this!